Public/Src/Pips/Dll/Operations/Process.cs

```diff
@@ -776,6 +790,7 @@ internal override void InternalSerialize(PipWriter writer)
             writer.Write((byte)ProcessAbsentPathProbeInUndeclaredOpaquesMode);
             writer.Write((byte)DoubleWritePolicy);
             writer.Write((byte)ContainerIsolationLevel);
+            writer.Write(Weight);

```

Comment:
Since it seems the weight is slots, perhaps use `WriteCompact`?

Reply:
and readCompact

---
In reply to: [271091377](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271091377) [](ancestors = 271091377)

Reply:
slots might be fn(#cores) which users wont know beforehand, so users might just want to set weight to some value like 9999999 to make sure a process runs solo.

---
In reply to: [271091402](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271091402) [](ancestors = 271091402,271091377)